encoder/vaapiencoder_h264.cpp

```diff
@@ -927,9 +919,18 @@ Encode_Status VaapiEncoderH264::reorder(const SurfacePtr& surface, uint64_t time
 
     PicturePtr picture(new VaapiEncPictureH264(m_context, surface, timeStamp));
 
-    bool isIdr = (m_frameIndex == 0 ||m_frameIndex >= keyFramePeriod() || forceKeyFrame);
+    bool isIdr = (m_frameIndex == 0 ||m_frameIndex >= m_keyPeriod || forceKeyFrame);
 
     if (isIdr) {
+        // If the last frame before IDR is B frame, set it to P frame.
```

Comment:
Yes, this fix has been merged to master already. But I don’t know why it shows here or how to remove it. Maybe it’s generated automatically by github? Could you advise how to deal with it?

Best Regards,
Xiaoxia Zhong

From: Xu Guangxin [mailto:notifications@github.com]
Sent: Monday, February 29, 2016 10:03 AM
To: 01org/libyami
Cc: Zhong, Xiaoxia
Subject: Re: [libyami] Add IDR interval setting to H.264 encoder (#447)


In encoder/vaapiencoder_h264.cpp<https://github.com/01org/libyami/pull/447#discussion_r54363332>:

>

>      if (isIdr) {

> +        // If the last frame before IDR is B frame, set it to P frame.

why still have this. seems it have been merged to master. https://github.com/01org/libyami/blob/master/encoder/vaapiencoder_h264.cpp#L934

—
Reply to this email directly or view it on GitHub<https://github.com/01org/libyami/pull/447/files#r54363332>.